src/components/Layout/index.tsx

```diff
@@ -37,42 +38,25 @@ const Layout: FC<Props> = ({ children }) => {
             {isLoading && <Loader />}
             <div className={styles.wrapper}>
                 <div className={styles.header}>
-                    {navBarContent('Tasks', '/', router.pathname === '/')}|
-                    {navBarContent(
-                        'Issues',
-                        '/issues',
-                        router.pathname === '/issues'
-                    )}
-                    |
-                    {navBarContent(
-                        'Mine',
-                        '/mine',
-                        router.pathname === '/mine'
-                    )}
-                    {/* TODO: Uncomment when DS(Chanllenges) is ready */}
-                    {/* |
-                    {navBarContent(
-                        'DS',
-                        '/challenges',
-                        router.pathname === '/challenges'
-                    )} */}
-                    |
-                    {navBarContent(
-                        'Open PRs',
-                        '/openPRs',
-                        router.pathname === '/openPRs'
-                    )}
-                    |
-                    {navBarContent(
-                        'Stale PRs',
-                        '/stale-pr',
-                        router.pathname === '/stale-pr'
-                    )}
-                    |
-                    {navBarContent(
-                        'Idle Users',
-                        '/idle-users',
-                        router.pathname === '/idle-users'
+                    {navBarContentMock.map((element, index) => (
+                        <React.Fragment key={index}>
+                            {navBarContent(
+                                element.title,
+                                element.refURL,
+                                router.pathname === element.pathName
+                            )}
+                            {element.pipeSymbol}
+                        </React.Fragment>
+                    ))}
+                    {dev && (
+                        <>
+                            |
+                            {navBarContent(
+                                'Standup',
+                                '/standup/?dev=true',
+                                router.pathname === '/standup'
+                            )}
+                        </>
```

Comment:
just like you did for the above links can you do the same here for the `dev=true` features?

Reply:
I think we have to hide only Standup links not added in the route like the Availability Panel.

Reply:
fixed